oshi-core/src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -74,6 +74,8 @@
     private final int major;
     private final int minor;
 
+    public static String MACOS_VERSIONS_PROPERTIES;
```

Comment:
Also need a `final` modifier here, and the full string declaration `= "oshi.macos.versions.properties";`